app/presenters/admin_access_presenter.rb

```diff
@@ -3,15 +3,22 @@
 class AdminAccessPresenter < SimpleDelegator
   include Memery
 
+  attr_reader :admin_access_level
   attr_reader :admin
 
   def initialize(admin)
-    @admin = admin
     super
+    @admin = admin
+    @admin_access_level = admin.access_level
   end
 
   def display_access_level
-    OpenStruct.new(UserAccess::LEVELS.fetch(admin.access_level.to_sym))
+    access_level = if admin_access_level
+      UserAccess::LEVELS.fetch(admin_access_level.to_sym)
+    else
+      {name: "(Not Set)", description: "N/A"}
+    end
+    OpenStruct.new(access_level)
```

Comment:
This was needed so that the admin listing page wouldn't blow up for users who don't have an access level. Found this while trying to kick the tires of perms v2 in dev.